src/Console/AskQuestionTrait.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Console;
+
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Question\Question;
+
+trait AskQuestionTrait
+{
+    /**
+     * @var QuestionHelper
+     */
+    protected $questionHelper;
+
+    protected function questionHelper()
+    {
+        if (is_null($this->questionHelper)) {
+            $this->questionHelper = new QuestionHelper;
+        }
+
+        return $this->questionHelper;
+    }
+
+    protected function ask($question, $default = null)
+    {
+        $question = new Question("<question>$question</question> ", $default);
+
+        return $this->questionHelper()->ask($this->input, $this->output, $question);
+    }
+
+    protected function secret($question)
+    {
+        $question = new Question("<question>$question</question> ");
+
+        $question->setHidden(true)->setHiddenFallback(true);
+
+        return $this->questionHelper()->ask($this->input, $this->output, $question);
+    }
+
+    protected function confirm($question)
+    {
+        while (true) {
+            $confirmation = $this->ask($question.' (y/N)');
```

Comment:
This seems to suggest that `n` is the default. But the subsequent code would just report "Invalid Response", if I hit Enter immediately, right?

Reply:
I don't think it does, my goal here is that unless the user enters y or n, it would keep asking. I know that I've hit the wrong key in these before by accident, resulting in something that wasn't the result I wanted, which is why I'm proposing to do it this way.

Reply:
That's a fine approach, but why is the `N` upper-case and the `y` lower-case?

Reply:
Not sure tbh, thought that was convention. Ill standardize.